server/helpers/source-content.js

```diff
@@ -7,7 +7,7 @@ const getPostsFromPath = (path) => {
   const files = fs.readdirSync(path);
   files.forEach((file) => {
     if (file.match(/.\.html$/)) {
-      let post ={};
+      let post ={ tags: []};
```

Comment:
Ensure a post has tags as haven't added meta data to all posts - should probably have full methods here.